content/docs/pipelines/deployment-alternatives.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "Pipelines Standalone Deployment Alternatives"
+description = "Alternative approaches to deploy Kubeflow Pipelines standalone to a cluster"
+weight = 15
++++
+
+As an alternative to [deploying Kubeflow](/docs/started/getting-started/#installing-kubeflow) as a
+whole with many components including pipelines, you also have a choice to deploy
+only Kubeflow Pipelines. Follow the instructions below to deploy
+Kubeflow Pipelines standalone using the supplied kustomize
+manifests.
+
+Knowledge about [Kubernetes](https://kubernetes.io/docs/home/), [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) and [kustomize](https://kustomize.io/) will help you understand this
+document better and be able to customize based on your needs.
+
+
+## Deploying Kubeflow Pipelines standalone to an existing cluster
+
+1. Deploy latest version of Kubeflow Pipelines:
+```
+export PIPELINE_VERSION={{% kfp-latest-version %}}
```

Comment:
who will be maintaining kfp-latest-version? 
alternatively we could publish the file to gs://ml-pipeline/pipeline-lite/latest/namespaced-install.yaml
and refer from there. 
wdyt?

Reply:
I think it's important we make the version explicit, it affects what command they use to delete it.

Reply:
+1 to making the version explicit, and to adding the step to the release book. (Doc updates should be part of each release, and part of each feature-development task.)

Reply:
discussed, I will add this to release book